src/ape/managers/chain.py

```diff
@@ -1154,6 +1154,9 @@ def instance_at(
         address: Union[str, AddressType],
         contract_type: Optional[ContractType] = None,
         txn_hash: Optional[str] = None,
+        abi: Optional[
+            Union[Union[List[ABI], object], str, Path]
```

Comment:
`object` should be `dict`?

Reply:
I got the error from mypy. theoretically it should be a dict

Reply:
fixed